config.yaml

```diff
@@ -0,0 +1,20 @@
+services:
```

Comment:
This looks better! I would probably aim for making this config file as minimal as possible though, as to not distract from the example. Would something like this be enough?

```yaml
services:
  play:
    url: https://play.openpolicyagent.org

bundles:
  authz:
    service: play
    resource: bundles/WfbLy2DbiF

decision_logs:
  console: true

default_decision: /http/example/authz/allow
```

Reply:
Yes, I just updated the config as your suggestion.